src/Persistance/Migration/v0.04-system-register-update/02-uri-domain.sql

```diff
@@ -1,2 +1,2 @@
 -- Domain: system_register.uri
-CREATE DOMAIN business_application.uri AS text CONSTRAINT uri_valid CHECK (value ~ '^https?:\/\/');
\ No newline at end of file
+CREATE DOMAIN business_application.uri AS text CONSTRAINT uri_startswith_https CHECK (value ~ '^https?:\/\/[^.]+');
```

Comment:
This constraint name is wrong, cause it doesn't check that it starts with https, it checks that it has http or https as scheme. Probably better to name it something like `uri_scheme_check`